data/sql/db-world/zone_difficulty_hardmode_ai.sql

```diff
@@ -0,0 +1,31 @@
+-- ************************************************************************
+-- Possible values for `Target`:
+SET @TARGET_SELF = 1;                       -- Self cast.
+SET @TARGET_VICTIM = 2;                     -- Our current target. (ie: highest aggro)
+SET @TARGET_HOSTILE_SECOND_AGGRO = 3;       -- Second highest aggro.
+SET @TARGET_HOSTILE_LAST_AGGRO = 4;         -- Lowest aggro.
+SET @TARGET_HOSTILE_RANDOM = 5;             -- Just any random player on our threat list.
+SET @TARGET_HOSTILE_RANDOM_NOT_TOP = 6;     -- Just any random player on our threat list except the current target.
+SET @TARGET_PLAYER_DISTANCE = 18;           -- All players in range. TargetArg = max range.
+-- ************************************************************************
+
+DROP TABLE IF EXISTS `zone_difficulty_hardmode_ai`;
+CREATE TABLE `zone_difficulty_hardmode_ai` (
+    `CreatureEntry` INT NOT NULL DEFAULT 0,
+    `Chance` TINYINT NOT NULL DEFAULT 100,          -- 0-100% chance for the creature to gain this spell
+    `Spell` INT NOT NULL,                           -- spell id
+    `Target` TINYINT NOT NULL DEFAULT 1,            -- see above
+    `TargetArg` TINYINT NOT NULL DEFAULT 0,         -- optional argument for the target. Max range for TARGET_PLAYER_RANGE
+    `Delay` INT NOT NULL DEFAULT 1,                 -- time in ms before first cast
+    `Cooldown` INT NOT NULL DEFAULT 1,              -- time in ms between casts
+    `Repetitions` TINYINT NOT NULL DEFAULT 0,       -- 0 = forever, 1 = just once. Room for future counters.
+    `Enabled` TINYINT DEFAULT 1,                    -- 0 = disabled, 1 = enabled
+	`Comment` TEXT
```

Comment:
Can this be NULL? Should it have a default value like NULL or ''?

Reply:
It's not read by the script. The only purpose is a brief comment on what it is supposed to do for those reading the SQL table.

Reply:
yea I know, my point was just to either add `TEXT NULL DEFAULT NULL` or `TEXT NOT NULL DEFAULT ''`
Because now it doesnt have any default values, if say you were to forget adding a comment